codegen/gql_build/lib/src/utils/config.dart

```diff
@@ -10,7 +10,7 @@ Map<String, Reference> typeOverrideMap(dynamic typeOverrideConfig) {
           entry.key as String,
           Reference(
             entry.value["name"] as String,
-            entry.value["import"] as String,
+            entry.value["import"] as String?,
```

Comment:
@knaeckeKami Please have a look here, since `entry.value["import"]` was null and I was facing issue with code gen, I made it nullable. I am not sure what it does and why it was not nullable type before.